openbas-front/src/public/components/channels/ChannelNewspaper.js

```diff
@@ -75,10 +85,27 @@ const ChannelNewspaper = ({ channelReader }) => {
         <div
           style={{ margin: '0 auto', textAlign: 'center', marginBottom: 15 }}
         >
-          <img
-            src={`${baseUri}/documents/${logo}/file${queryParams}`}
-            className={classes.logo}
-          />
+          <Tooltip title={channel?.channel_name}>
```

Comment:
Wouldn't it be better to have an if else?
I suppose, sometimes we have the logo to display

Reply:
I cancelled this change. As seen together, this is a config problem